src/net.cpp

```diff
@@ -2672,6 +2672,11 @@ ServiceFlags CConnman::GetLocalServices() const
     return nLocalServices;
 }
 
+void CConnman::SetLocalServices(ServiceFlags flagsIn)
```

Comment:
I'm entirely unconvinced NODE_NETWORK_LIMITED needs this (and would prefer to avoid adding it until we need it, though its no big deal to add it).